src/main/java/ru/skypro/homework/entity/User.java

```diff
@@ -0,0 +1,50 @@
+package ru.skypro.homework.entity;
+
+import lombok.*;
+import org.hibernate.Hibernate;
+import ru.skypro.homework.dto.Role;
+import javax.persistence.Entity;
```

Comment:
может вместо javax использовать jakarta?
```suggestion
import jakarta.persistence.*;
```

Reply:
У нас спринг 2.7 чего-то там в нет нет jakart'ы